Assets/Scripts/Managers/Stats.cs

```diff
@@ -144,6 +152,7 @@ private void OnNextTurnBegin()
                 Stability = 100;
             }
             Wealth += WealthPerTurn;
+            TotalWealth += WealthPerTurn;
```

Comment:
Does this run on turn 0?

Reply:
Nope, its only triggered on entering the 'NextTurn' state, so wont happen at the start of the game